xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTickerFundingCurrency.java

```diff
@@ -36,4 +38,83 @@
   public boolean isFundingCurrency() {
     return true;
   }
+
+  @Override
+  public String getSymbol() {
+    return symbol;
+  }
```

Comment:
lombok is used here, this code is not needed

Reply:
Ok will remove these :)